client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -795,16 +795,34 @@ export class DomainWarnings extends React.PureComponent {
 		switch ( domainInTransfer.transferStatus ) {
 			case transferStatus.PENDING_OWNER:
 				compactMessage = translate( 'Transfer confirmation required' );
-				message = translate(
-					'We sent an email to confirm the transfer of {{strong}}%(domain)s{{/strong}}. {{a}}More Info{{/a}}',
-					{
-						components: {
-							strong: <strong />,
-							a: <a href={ domainManagementLink } rel="noopener noreferrer" />,
-						},
-						args: { domain: domainInTransfer.name },
-					}
-				);
+
+				const translateParams = {
+					components: {
+						strong: <strong />,
+						a: <a href={ domainManagementLink } rel="noopener noreferrer" />,
+					},
+					args: { domain: domainInTransfer.name },
```

Comment:
Do we also need `email: domainInTransfer.email` here for the case where we have the email address to show?

Reply:
Nope. I set it explicitly for that message https://github.com/Automattic/wp-calypso/pull/21470/files#diff-ef7444613272677858192c00d814e81cR814.

Reply:
Ha, I guess I should've looked harder there. Nice work!